pkg/cli/image/mirror/plan.go

```diff
@@ -615,7 +614,7 @@ type repositoryManifestPlan struct {
 	}
 }
 
-func (p *repositoryManifestPlan) Copy(srcDigest godigest.Digest, srcManifest distribution.Manifest, tags []string, to distribution.ManifestService, toBlobs distribution.BlobService) {
+func (p *repositoryManifestPlan) Copy(srcDigest godigest.Digest, srcManifest distribution.Manifest, prerequisites []godigest.Digest, tags []string, to distribution.ManifestService, toBlobs distribution.BlobService) {
```

Comment:
Now manifests not in `manifestlist.DeserializedManifestList` type should pass `prerequisites` as nil?